Games/LLEGSB/script.js

```diff
@@ -0,0 +1,216 @@
+  var b = 2;
+  var c = Math.pow(a, b)
+  questionText.text = `Which of the following values of x is a solution of the equation given: x^2 = ${c}`;
+  correctAnswer = `${a}`;
+}
+
+//question 3 function
+function question3(){
+  document.getElementById('demo').innerHTML = "Question 3";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = ''
+  var a = Math.floor(Math.random() * (70 - 20 + 1)) + 20;
+  var b = Math.round(a*0.15);
+  questionText.text = `A waiter received a 15% tip for a restaurant bill of $${a}. Estimate the tip the waiter received to the nearest dollar($).`
+  correctAnswer = `$${b}`
+}
+
+// question 4 function
+// like terms question
+function question4(){
+  document.getElementById('demo').innerHTML = "Question 4";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = ''
+  var a = Math.floor(Math.random() * (-20 - 20 + 1)) + 20;
+  var b = Math.floor(Math.random() * (10 - 2 + 1)) + 2;
+  var c = Math.floor(Math.random() * (7 - 2 + 1)) + 2;
+  var d = Math.floor(Math.random() * (-20 - 20 + 1)) + 20;
+  var e = Math.floor(Math.random() * (10 - 2 + 1)) + 2;
+  questionText.text = `Combine all like terms in the given expression: ${a}x + ${b}y + ${d}x - ${c}z + ${e}y` 
+  correctAnswer = `${a+d}x+${b+e}y-${c}z`;
+}
+
+// question 5 function
+// equation of a line
+function question5(){
+  document.getElementById('demo').innerHTML = "Question 5";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = ''
+  var y1 = Math.floor(Math.random() * (200 - 1 + 1)) + 1;
+  var y2 = Math.floor(Math.random() * (200 - 1 + 1)) + 1;
+  var x1 = Math.floor(Math.random() * (200 - 1 + 1)) + 1;
+  var x2 = Math.floor(Math.random() * (200 - 1 + 1)) + 1;
+  questionText.text = `What is the slope of a line that contain the points (${x1}, ${y1}) and (${x2}, ${y2}).`;
+  var slopeEquation = ((y2-y1)/(x2-x1));
+  slopeRounded = slopeEquation.toFixed(2);
+  correctAnswer = `${slopeRounded}`;
+}
+
+function question6(){
+  document.getElementById('demo').innerHTML = "Question 6";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = ''
+  var x1 = Math.floor(Math.random() * (300 - 100 + 1)) + 100;
+  var y1 = Math.floor(Math.random() * (300 - 100 + 1)) + 100;
+  var x2 = Math.floor(Math.random() * (300 - 100 + 1)) + 100;
+  var y2 = Math.floor(Math.random() * (300 - 100 + 1)) + 100;
+  questionText.text = `What is the length of a line segment with the endpoints (${x1}, ${y1}) and (${x2}, ${y2})`  
+  var x = (x2-x1); 
+  var y = (y2-y2);
+  var xSquare = (x**2);
+  var ySquare = (y**2);
+  var add = (xSquare+ySquare);
+  var squareRoot = (add**0.5);
+  ansRounded = squareRoot.toFixed(2);
+  correctAnswer = ansRounded;
+  console.log(correctAnswer)
+}
+//
+// a function to check the answer for question 
+
+function checkAnswer(answer){
+  hintText.text = '';
+    if (answer == correctAnswer)
+      (confirmationText.text = 'Correct!')
```

Comment:
You don't need to wrap these statements with parenthesis.
You need to put curly braces around this block of code to include in the execution if the condition is true.
Eg.
```js
if (something) {
   statement1;
   statement2;
   etc.
}
else {
   stuff;
   stuff;
}
```